plugin.yaml

```diff
@@ -10,7 +10,7 @@ plugins:
     # URL to archive containing the plugin or name of directory containing
     # the plugin if it is included in the the blueprint directory under the
     # "plugins" directory.
-    source: https://github.com/szpotona/cloudify-gcp-plugin/archive/devel.zip
+    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/master.zip
```

Comment:
I think we should point at a specific version - like a tag name - here instead of at master because master is frequently changing...